CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ include("CMake/AddDependencies.cmake")
 include("CMake/Fixups.cmake")
 include("CMake/GenerateBindings.cmake")
 include("CMake/GroupSources.cmake")
+include("CMake/SetVersion.cmake")
```

Comment:
I believe my contribution to this file does not meet the creativity requirement to receive copyright.